inc/saiswitch.h

```diff
@@ -3518,6 +3518,12 @@ typedef enum _sai_switch_asic_sdk_health_category_t
  */
 typedef enum _sai_switch_stat_t
 {
+    /** Switch stat range start */
+    SAI_SWITCH_STAT_START,
+
+    /** Packets trimmed but dropped due to failed shared buffer admission on a new queue */
+    SAI_SWITCH_STAT_TRIM_DROPPED_PACKETS = SAI_SWITCH_STAT_START,
```

Comment:
please follow concept on other ranges if that the range name
```
SAI_SWITCH_STAT_TRIM_DROPPED_PACKETS_RANGE_BASE = SAI_SWITCH_STAT_START
...
SAI_SWITCH_STAT_TRIM_DROPPED_PACKETS_RANGE_END = 0x00000fff
```

or introduce other range, are you sure your stat type dont fit any existing range ?

Reply:
the existing switch stats belong to clear group - in drop reasons (Debug counters), out drop reasons (debug counters) or fabric
While the trim stats are "regular" group
I don't see we keep regular group in other places like port or queue, nor do we group every stat to a group
in the future we may add other regular stats